src/main/java/algorithms/QuickSelect.java

```diff
@@ -0,0 +1,73 @@
+package algorithms;
+
+import java.util.Arrays;
+
+public class Quickselect {
```

Comment:
```suggestion
public class QuickSelect{
```
In order to solve the error message.

Reply:
File's name is wrong.